packages/specs/src/openapi.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.1
+openapi: 3.1.0
```

Comment:
This should be updated in a separate PR
```suggestion
openapi: 3.0.1
```

Reply:
Hmmm perhaps, i updated it to satisfy the openapi spec version regex 🤔 
![image](https://github.com/directus/directus/assets/9389634/1c8e8ddd-892c-4e52-b9e3-645b696c6a89)

Reply:
Yeah, we should update (and probably fix that) soon 😄

Reply:
A follow up PR for that it is 👌